drinks_touch/elements/elm_list.py

```diff
@@ -22,12 +22,9 @@ def __update_elements(self):
         if len(self.elements) > self.max_elm_count:
             self.elements.pop(0)
             for e in self.elements:
-                e.pos = (
-                    self.pos[0],
-                    e.pos[1] - (e.height + self.elm_margin)
-                )
+                e.pos = (self.pos[0], e.pos[1] - (e.height + self.elm_margin))
         else:
-            last_elm = self.elements[len(self.elements) - 1:][0]
+            last_elm = self.elements[-1]
```

Comment:
Manual change, please review

Reply:
lol, that code was horrible… thanks for fixing :D :+1: